dev.alextren.Spot.svg

```diff
@@ -0,0 +1,236 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Looks like this file is not being used now in the manifest, can you remove it?

Reply:
removed it!